test/Microsoft.AspNetCore.SignalR.Client.Tests/HubConnectionTests.Protocol.cs

```diff
@@ -63,6 +63,25 @@ public class Protocol
                 }
             }
 
+            [Fact]
+            public async Task ClientIsOkayReceivingMinorVersionInHandshake()
+            {
+                var connection = new TestConnection(autoHandshake: false);
+                var hubConnection = CreateHubConnection(connection);
+                try
+                {
+                    var startTask = hubConnection.StartAsync();
+                    var message = await connection.ReadHandshakeAndSendResponseAsync(56);
+
+                    await startTask;
```

Comment:
Add a comment here: "We're just testing that the client doesn't fail when a minor version is added to the handshake. The client doesn't actually use that version anywhere yet so there's nothing else to test at this time"